SimpleNotePad.java

```diff
@@ -0,0 +1,437 @@
+
+import java.nio.file.Path;
+
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+
+import java.util.HashMap;
+
+import java.util.Scanner;
+
+
+
+
+
+public class SimpleNotePad extends JFrame implements ActionListener{
+
+	// Creating my Arraylist to keep the files 
+
+	ArrayList<JMenuItem> rFile =new ArrayList<JMenuItem>();
+
+	
+
+	// Creating button
+	JTextArea textInput;
+
+	JMenuBar barMenu = new JMenuBar();
+
+	JMenu fileMenu = new JMenu("File");  
+
+	JMenu editMenu = new JMenu("Edit");
+
+	JMenu openRecent = new JMenu("Recent");
+
+	JTextPane noteArea = new JTextPane(); 
+
+	JMenuItem openFileMenu = new JMenuItem("Open");
+	
+	JMenuItem printFileMenu = new JMenuItem("Print File"); 
+	
+	JMenuItem saveFileMenu = new JMenuItem("Save File"); 
+	
+	JMenuItem newFileMenu = new JMenuItem("New File"); 
+
+
+	JMenuItem copyMenu = new JMenuItem("Copy"); 
+	
+	JMenuItem pasteMenu = new JMenuItem("Paste");
+	
+
+	JMenuItem ReplaceMenu = new JMenuItem("replace");
+
+
+
+
+
+
+	private Object textArea;
+
+
+
+	private SimpleNotePad() {
+
+		setTitle("A Simple Notepad Tool");
+
+		fileMenu.add(openFileMenu);
+		
+		fileMenu.addSeparator();
+		
+		fileMenu.add(printFileMenu);
+		
+		fileMenu.addSeparator();
+		
+		fileMenu.add(saveFileMenu);
+		
+		fileMenu.addSeparator();
+		
+		fileMenu.add(newFileMenu);
+		
+		fileMenu.addSeparator();  
+		
+		fileMenu.add(openRecent); 
+
+		fileMenu.addSeparator();  
+
+
+		//editMenu.add(undoMenu);
+		editMenu.add(ReplaceMenu);
+
+		editMenu.add(pasteMenu);
+		
+		editMenu.add(copyMenu);
+
+
+
+		newFileMenu.addActionListener(this);	
+
+		newFileMenu.setActionCommand("new");
+
+		saveFileMenu.addActionListener(this);
+
+		saveFileMenu.setActionCommand("save");
+
+		printFileMenu.addActionListener(this);
+
+		printFileMenu.setActionCommand("print");
+
+		copyMenu.addActionListener(this);
+
+		copyMenu.setActionCommand("copy");
+
+		pasteMenu.addActionListener(this);
+
+		pasteMenu.setActionCommand("paste");
+
+		openFileMenu.addActionListener(this);
+
+		openFileMenu.setActionCommand("Open File");
+
+		openRecent.addActionListener(this); 
+
+		openRecent.setActionCommand("Recent");
+
+		ReplaceMenu.addActionListener(this); 
+
+		ReplaceMenu.setActionCommand("replace");
+
+
+
+
+
+		barMenu.add(fileMenu);
+
+		barMenu.add(editMenu);
+
+
+
+		setJMenuBar(barMenu);
+
+
+
+		add(new JScrollPane(noteArea));
+
+
+
+		setPreferredSize(new Dimension(600,600));
+
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+
+		setVisible(true);
+
+
+
+		pack();	
+
+	}
+
+
+
+	public static void main(String[] args) {
+
+		SimpleNotePad app = new SimpleNotePad();
+
+	}
+
+	private void pasteImplementation() {
+
+		StyledDocument doc = noteArea.getStyledDocument();
+
+		//Position position = doc.getEndPosition();
+
+		noteArea.paste();	
+
+	}
+
+
+
+	@Override
+
+	// EveryTime there is a button clicked , it checks here
+
+	public void actionPerformed(ActionEvent event) {
+
+		if(event.getActionCommand().equals("new")) {

```

Comment:
Consider splitting this up into different methods so there is not such a long if/else statement. Or consider fixing it as you would a switch.